recipes/gmm/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# from https://web.archive.org/web/20200225211624/http://getfem.org/gmm/install.html
+# configure first using conda prefix
+./configure --prefix=${PREFIX}
+# make it
```

Comment:
These comments don't seem helpful 🤣

Reply:
yeah this one isn't I suppose, but as a person who doesn't use the c/c++ stack of cmake/autotools/gcc/make day-to-day some of the earlier ones are more helpful for me personally. Conda-forge could use more docs for contributing c/c++ packages in general (or at least link to conda-build more). Obvious to some but not to all.